src/Logging.XUnit/XUnitLogger.cs

```diff
@@ -264,28 +267,53 @@ private static string GetLogLevelString(LogLevel logLevel)
         private static void GetScopeInformation(StringBuilder builder)
         {
             var current = XUnitLogScope.Current;
-            string scopeLog;
-            int length = builder.Length;
 
+            var stack = new Stack<XUnitLogScope>();
             while (current != null)
             {
-                if (length == builder.Length)
-                {
-                    scopeLog = $"=> {current}";
-                }
-                else
+                stack.Push(current);
+                current = current.Parent;
+            }
+
+            var depth = 0;
+            string DepthPadding(int depth) => new string(' ', depth * 2);
+
+            while (stack.Any())
+            {
+                var elem = stack.Pop();
+                foreach (var property in StringifyScope(elem))
                 {
-                    scopeLog = $"=> {current} ";
+                    builder.AppendLine($"{MessagePadding}{DepthPadding(depth)}=> {property}");
```

Comment:
You can leverage the `StringBuilder` here to avoid the interpolation:

```suggestion
                    builder.Append(MessagePadding)
                           .Append(DepthPadding(depth))
                           .Append("=> ")
                           .AppendLine(property);
```